Client/Frontend/Library/RecentlyClosedTabsPanel.swift

```diff
@@ -56,6 +56,14 @@ class RecentlyClosedTabsPanel: UIViewController, LibraryPanel {
             tableViewController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor)
         ])
     }
+
+    override func viewDidAppear(_ animated: Bool) {
+        guard recentlyClosedTabsDelegate != nil else {
+            recentlyClosedTabsDelegate = sceneForVC?.browserViewController
+
+            return
+        }
+    }
```

Comment:
I believe you are missing the super statement here. Its good practice to add it. 

```suggestion
    override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(animated)
        guard recentlyClosedTabsDelegate != nil else {
            recentlyClosedTabsDelegate = sceneForVC?.browserViewController
            return
        }
    }
```

Reply:
Would be good to add a note here `RecentlyClosedTabsPanel` is `AnyObject` and why you are assigning BVC from the current valid scene to it so that in future we know why we did it.